src/c++/perf_analyzer/data_loader.cc

```diff
@@ -90,8 +90,8 @@ DataLoader::ReadDataFromDir(
       if (input_string_data.size() != batch1_num_strings) {
         return cb::Error(
             "provided data for input " + input.second.name_ + " has " +
-                std::to_string(it->second.size()) + " byte elements, expect " +
-                std::to_string(batch1_num_strings),
+                std::to_string(input_string_data.size()) +
```

Comment:
This was a bad error message. If I supplied 2 strings when it expected 1, it would say "You provided 21 bytes, expected 1". Now it will say "You provided 2 elements. Expected 1", which matches what the actual check is and actually makes sense to the user